.github/workflows/build.yml

```
@@ -130,6 +130,22 @@ jobs:
               comment_id: +COMMENT_ID,
             } );
 
+      - name: Trigger TeamCity build
+        if: ${{ fromJSON(steps.changed.outputs.projects)['plugins/jetpack'] == true }}
+        env:
+          SECRET: ${{ secrets.JETPACK__GITHUB_TEAMCITY_TOKEN }}
+          URL: ${{ secrets.TEAMCITY_TRIGGER_ENDPOINT }}
+          PR: ${{ github.event.number }}
+          SHA: ${{ github.event.pull_request.head.sha }}
```

One easy way to fix the trunk-push issue might be to do like this.
```suggestion
          PR: ${{ github.event_name == 'push' && github.ref_name || github.event.number }}
          SHA: ${{ github.event_name == 'push' && github.sha || github.event.pull_request.head.sha }}
```
That would result in "pr-number" being "trunk" (or possibly something like "jetpack/branch-11.5") for pushes.